client/app/assets/styles/variables.js

```diff
@@ -113,4 +123,24 @@ module.exports = {
   '--SearchBar_mobileTriangleSize': '8px',
   '--SearchBar_iconSize': '16px',
   '--SearchBar_iconTopMarginFix': '4px',
+
+  '--MenuItem_backgroundColorHover': backgroundLightColorHover,
+  '--MenuItem_borderColor': borderColor,
+  '--MenuItem_paddingTopbarVertical': paddingTopbarVertical,
+  '--MenuItem_paddingTopbarHorizontal': paddingTopbarHorizontal,
+  '--MenuItem_textColor': textColor,
+  '--MenuItem_textColorFocus': textColorFocus,
+  '--MenuItem_letterSpacing': '0.09px',
```

Comment:
Janne, defines letter spacings quite much per font & font-size. It might be that these are not relative, but are based on gut feeling case by case. I'd leave it as it is - and think about these after deadline.